wallet/src/de/schildbach/wallet/ui/TransactionResultActivity.kt

```diff
@@ -75,7 +86,9 @@ class TransactionResultActivity : AbstractWalletActivity() {
         val txId = intent.getSerializableExtra(EXTRA_TX_ID) as Sha256Hash
         setContentView(R.layout.activity_successful_transaction)
 
-        val transactionResultViewBinder = TransactionResultViewBinder(container)
+        val blockchainIdentity: BlockchainIdentity? = runBlocking { PlatformRepo.getInstance().getBlockchainIdentity() }
```

Comment:
Should we use an observer instead?  perhaps a LiveData that returns a BlockchainIdentity object?